modules/cognitiveservices/multi/main.bicep

```diff
@@ -0,0 +1,172 @@
+@description('The location into which your Azure resources should be deployed.')
+param location string = resourceGroup().location
+
+@description('The kind of Cognitive Service to create. See: https://learn.microsoft.com/en-us/azure/cognitive-services/create-account-bicep for available kinds.')
+param kind string
```

Comment:
can we switch this term to "prefix" to follow the other templates, and provide a default value (ill link the website that has the mapping of default names)

Reply:
mmm... not sure that feels right...

This `kind` here maps to the kind of Cognitive Service we'll create. It's more significant than just naming.

The `prefix` defined in https://github.com/Azure/bicep-registry-modules/blob/main/CONTRIBUTING.md#mainbicep serves a different purpose.

If anything I think I could _add_ `prefix` `param` to this but I wouldn't switch it for `kind`.